src/Repository/MenuRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz' menu plugin for Sylius.
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusMenuPlugin\Repository;
+
+use MonsieurBiz\SyliusMenuPlugin\Entity\MenuInterface;
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+
+class MenuRepository extends EntityRepository implements MenuRepositoryInterface
+{
+    public function findByCode(string $localeCode, string $code): ?MenuInterface
```

Comment:
```suggestion
    public function findByCodeAndLocale(string $code, string $localeCode): ?MenuInterface
```

Naming the method like this maybe ?